### ISSUE TITLE
Market Maker - no oracles available for DCR/LTC

### ISSUE THREAD

**Issue by bochinchero** [decred/dcrdex#2334]
Fully aware that the MM bot is POC at this point, but I imagine this will be an issue on pretty much any market that doesn't already exist in a CEX. When the market doesn't have too much activity on the books (like DCR/LTC now) the basis price can drift significantly from where it should be (e.g. right now its 0.17 and in binance its ~0.20), oracles would really come in handy here.

Could this be calculated based on thbe USD or BTC prices of both assets on the pair?

**Comment by martonp** [comment]
Yes, if there is no oracle available, then the USD prices will be used to calculate a rate.

**Comment by chappjc** [comment]
coingecko and probably other sources can give conversions for arbitrary pairs.

```
$ curl -X 'GET' 'https://api.coingecko.com/api/v3/simple/price?ids=decred&vs_currencies=ltc' -H 'accept: application/json'
{"decred":{"ltc":0.20197442}}
```
